scripts/zkai_attention_kv_fused_softmax_table_section_delta_gate.py

```
@@ -0,0 +1,887 @@
+        "section_delta_bytes",
+        "bucket_delta_bytes",
+        "json_wrapper_delta_bytes",
+        "largest_delta_section",
+        "largest_delta_section_bytes",
+        "backend_internal_attribution",
+        "backend_internal_split_status",
+    }
+    if set(row) != expected:
+        raise FusedSoftmaxTableSectionDeltaGateError("section delta row field drift")
+    if row["profile_id"] not in EXPECTED_PROFILE_IDS:
+        raise FusedSoftmaxTableSectionDeltaGateError("profile_id drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)